tasks/broad/JointGenotypingTasks.wdl

```diff
@@ -273,6 +273,7 @@ task HardFilterAndMakeSitesOnlyVcf {
 
     String variant_filtered_vcf_filename
     String sites_only_vcf_filename
+    File? targets_interval_list # filters out all variants outside of the targets interval list for targetted sequencing
```

Comment:
`targetted` -> `targeted`.

I'm also unclear about whether such changes to tasks need to be reflected in a changelog somewhere?